infra/main.tf

```diff
@@ -28,9 +29,10 @@ module "env_prod" {
   source    = "git::ssh://git@github.com/futurice/symptomradar.git//infra/modules/main?ref=v2.3"
   providers = { aws.us_east_1 = aws.us_east_1 } # this alias is needed because ACM is only available in the "us-east-1" region
 
-  name_prefix          = "${var.name_prefix}-prod"
-  tags                 = merge(var.tags, { Environment = "prod" })
-  frontend_domain      = "www.oiretutka.fi"
+  name_prefix     = "${var.name_prefix}-prod"
+  tags            = merge(var.tags, { Environment = "prod" })
+  frontend_domain = "www.oiretutka.fi"
+  # TODO: apex_redirect_domain = "oiretutka.fi"
```

Comment:
I see this commented out, so is this or is this not still performing the redirect on prod?

Reply:
Yeah, that's correct!

Because our prod infra is pinned to the previous release, it means it won't be updated willy-nilly, only during the release process. As a consequence, prod infra doesn't have this feature until the next release, and thus it doesn't (yet) support the `apex_redirect_domain` input in its instance of the `main` module.

Yeah, it's a bit confusing, but I just left that comment there so that during the next release we don't need to wonder what value we should add for this input, as Terraform will start asking for it when we release the version pin for prod. :)